apps/hpc-cdm/src/app/pages/admin.tsx

```diff
@@ -12,44 +12,50 @@ import PageMeta from '../components/page-meta';
 const PageAdmin = () => {
   return (
     <AppContext.Consumer>
-      {({ lang }) => (
-        <div className={CLASSES.CONTAINER.CENTERED}>
-          <PageMeta title={[t.t(lang, (s) => s.navigation.admin)]} />
-          <C.SidebarNavigation
-            menu={[
-              {
-                label: t.t(lang, (s) => s.navigation.manageAccess),
-                path: paths.adminAccess(),
-              },
-            ]}
-          >
-            <Routes>
-              <Route
-                path={paths.home()}
-                element={<Navigate to={paths.access()} />}
-              />
-              <Route
-                path={paths.access()}
-                element={
-                  <TargetAccessManagement
-                    target={{
-                      type: 'global',
-                    }}
-                  />
-                }
-              />
-              <Route
-                path={paths.root()}
-                element={
-                  <C.NotFound
-                    strings={t.get(lang, (s) => s.components.notFound)}
-                  />
-                }
-              />
-            </Routes>
-          </C.SidebarNavigation>
-        </div>
-      )}
+      {({ lang, access }) => {
+        const { canModifyGlobalUserAccess } = access().permissions;
+        if (!canModifyGlobalUserAccess) {
+          return <Navigate to={paths.home()} />;
+        }
```

Comment:
Previously, when `canModifyGlobalUserAccess` is `false`, user was redirected to `PageNotFound`, so let's do that again and not redirect to home page.